roles/installer/tasks/database_configuration.yml

```diff
@@ -78,6 +78,7 @@
     awx_postgres_database: "{{ pg_config['resources'][0]['data']['database'] | b64decode }}"
     awx_postgres_port: "{{ pg_config['resources'][0]['data']['port'] | b64decode }}"
     awx_postgres_host: "{{ pg_config['resources'][0]['data']['host'] | b64decode }}"
+    awx_postgres_sslmode: "{{ pg_config['resources'][0]['data']['sslmode'] | b64decode }}"
```

Comment:
nit: If the entry does not exist. Would that fail saying keyError `sslmode` does not exist ?

Reply:
Yes it would. So I've updated the PR to ensure this scenario. Tested with and without the `sslmode`  in the secret and worked as expected.